pkg/eval/unix/unix.go

```diff
@@ -0,0 +1,11 @@
+// Package `unix` exposes variables and functions that deal with features
+// unique to UNIX like operating systems. On other operating systems it will
```

Comment:
* UNIX like -> UNIX-like

* Let's make up our mind and just say non-UNIX operating systems will see an empty namespace.